project/AppDependencies.scala

```diff
@@ -6,9 +6,9 @@ object AppDependencies {
   val compile = Seq(
     "uk.gov.hmrc" %% "bootstrap-play-26" % "0.35.0",
```

Comment:
Not sure if I spotted this before. But this has a newer version
https://github.com/hmrc/bootstrap-play-26/releases